config.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+/* eslint no-process-env: 0 */
+module.exports = {
+  sql: {
+    host: process.env.SQL_HOST,
+    user: process.env.SQL_USER,
+    password: process.env.SQL_PASSWORD,
+    db: 'digitalInclusion',
+    table: 'research_data'
```

Comment:
This could probably just be a const elsewhere in the app. (Typically just in one file.)

Reply:
also, this is in config, but you don't refer to it anywhere?
You use the literal string `research_data` in the sql statement.

Reply:
yeah good point @JoeChapman